src/mutator/DescriptionMutator.ts

```diff
@@ -3,25 +3,27 @@ import { markAsAlreadyProcessed, textToImage } from './MutatorUtil';
 
 export const DescriptionMutator = {
   async embedPlantUmlImages(finders: Finder[], webPageUrl: string, $root: JQuery<Node>) {
-    for (const finder of finders) {
-      for (const content of finder.find(webPageUrl, $root)) {
-        const $text = content.$text;
+    const arraysOfContents = await Promise.all(finders.map((finder) => finder.find(webPageUrl, $root)));
```

Comment:
This statement waits all the promises, so I feel it is not good performance-wise.
Could we rewrite this as follows?

```
await Promise.all(finders.map(async (finder) => {
  const contents = await finder.find(webPageUrl, $root);
  for (const content of contents) {
    .. snip ..
  }
}));
```

Reply:
Shall I rewrite DiffMutator, too?

Reply:
yes, please